### DISCUSSION TITLE
How do I select a QTable row from JavaScript

### DISCUSSION THREAD

**Discussion by afx919** [quasarframework/quasar#15857] [General - Components / Directives / etc]
I would like to select a row by double-clicking it, but QTable does not provide methods and the @row-dblclick parameter has no props to modify

**Comment by pdanpdan** [comment]
What props to modify?

**Reply by afx919** [reply]
![image](https://github.com/quasarframework/quasar/assets/32212573/435e7e2c-546f-4da4-8a48-5754fa696a05)

like this, I want to implement a double click to change the Item select status,but QTable @row-dblclick event unable to access  'props.selected' property, There is also no method modification select status, only an 'isRowSelected' method and no 'setRowSelected'

**Comment by metalsadman** [comment]
![image](https://github.com/quasarframework/quasar/assets/31038838/a844535b-faed-4806-9077-424595b905a2)

**Reply by afx919** [reply]
As the title says，no 'props' parameter to modify the selection state, evt is JS event object, row is row data,index is row index.  no 'selected' property, how to change select status in 'row-dblclick' event?
